hack/make-rules/verify/misspell.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../../.." && pwd -P)"
+cd $REPO_ROOT
+
+BIN_DIR="${REPO_ROOT}/_bin/misspell"
+mkdir -p "$BIN_DIR"
+MISSPELL="${BIN_DIR}/misspell"
+
+echo "Ensuring go version."
+source ./hack/build/setup-go.sh
+
+# build misspell
+echo "Install misspell."
+cd "hack/tools"
+go build -o "$MISSPELL" github.com/client9/misspell/cmd/misspell
+cd "${REPO_ROOT}"
+
+trap 'echo ERROR: found unexpected instance of "Git"hub, use github or GitHub' ERR
```

Comment:
nit: why the quotes around Git here? we've actually found an instance of `Github` (no quotes) right?

Reply:
yeah the test itself would fail without quotes

Reply:
lol! in kubernetes we use `tac` to solve this 🙃